src/order/filter/basic.lean

```diff
@@ -893,9 +893,9 @@ begin
   { rw [finset.infi_insert, finset.set_bInter_insert, hs, inf_principal] },
 end
 
-@[simp] lemma infi_principal_fintype {ι : Type w} [fintype ι] (f : ι → set α) :
+@[simp] lemma infi_principal {ι : Type w} [finite ι] (f : ι → set α) :
```

Comment:
Any reason for removing `of_finite`?

Reply:
because `infi_principal_finite` already exists. I'm following the same convention as between `eventually_all` and `eventually_all_finite`.